Makefile.rules

```diff
@@ -290,7 +290,7 @@ endif
 DEPFILES = $(patsubst %,gen/deps/%.d,$(SOURCES))
 
 # Include the dependency tracking files, skip any missing ones
```

Comment:
Note that the comment already that we "skip any missing ones", but that wasn't true anymore with my previous refactoring of the buildsystem; this PR restores the behavior described in the comment